src/entities/Genre.entity.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  ManyToMany,
+  BaseEntity,
+} from 'typeorm';
+import { Song } from './Song.entity';
+
+@Entity()
+export class Genre extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  id!: number;
+
+  @Column({ nullable: false })
```

Comment:
```suggestion
  @Column()
```
`{ nullable: false }` -> mặc định sẽ là field required, nếu trg nào là optional thì e mới bật `true` nha